agac_torch/requirements.txt

```diff
@@ -0,0 +1,15 @@
+dataclasses==0.6
+gym~=0.18.0
+gym-minigrid==1.0.1
+joblib==0.16.0
+lxml==4.5.2
+mpi4py==3.0.3
+mujoco-py==2.0.2.11
+multidict==5.0.0
+neptune-client==0.9.8
+numpy~=1.18.5
+pre-commit==2.9.2
```

Comment:
Same (naive) question: what is the use of pre-commit here?

Reply:
Ditto !